Seeding external recipes

sheetaljoshi opened the pull request:
Seeding all external recipes across different content stores.

sheetaljoshi committed:
Organizing external content

sheetaljoshi committed:
Added section for Amazon Elasticsearch Service and also a section for Databases

sheetaljoshi committed:
Fixed formatting

sheetaljoshi committed:
Added AMG, AMP, Infra and Data

sheetaljoshi committed:
Added Observability Workshop referenecs

sheetaljoshi committed:
Fixed workshops section formatting

sheetaljoshi committed:
Fixed data page title

sheetaljoshi committed:
Added recipes to Cloudwatch

sheetaljoshi committed:
Changed troubleshoot to triage

sheetaljoshi committed:
Added Observability workshop links

mhausenblas commented:
Thanks a lot! This looks great. One thing: the new category `databases` you created is already present under `infra & data` (see for example RDS). Can you please remove it/merge with the existing category?

sheetaljoshi commented:
> Thanks a lot! This looks great. One thing: the new category `databases` you created is already present under `infra & data` (see for example RDS). Can you please remove it/merge with the existing category?

Acknowledged, I will change database section.

sheetaljoshi committed:
Removed separate database section and merging it under infra & data.

sheetaljoshi commented:
Removed database section.

sheetaljoshi committed:
Removed Database section

sheetaljoshi committed:
- added tasks sections
- moved rds and dynamodb from under data to their own sections
- moved workshops under tasks

mhausenblas commented:
Excellent, thanks a lot! This addresses the issue #2.
